package.json

```diff
@@ -20,14 +32,13 @@
     "@babel/code-frame": "^7.15.8",
     "chalk": "^4.1.2",
     "create-jest-runner": "^0.9.0",
-    "graceful-fs": "^4.2.8",
-    "jest-docblock": "^27.0.6",
-    "mlh-tsd": "^0.14.1"
+    "tsd-lite": "^0.3.0"
   },
   "devDependencies": {
     "@babel/core": "^7.15.8",
     "@babel/preset-env": "^7.15.8",
     "@babel/preset-typescript": "^7.15.0",
+    "@tsd/typescript": "^4.5.4",
```

Comment:
Yarn v2 cannot install `typescript` 4.5.4, but looks like it can install `@tsd/typescript` 4.5.4. Hm.. I think that’s fine. Or?